[20.943880081] 2018-06-21 21:09:13,145 - INFO - launcher: Starting sanity checks
[20.943880081] 2018-06-21 21:09:13,145 - INFO - launcher: Checking for Python 3.5+
[21.940946579] 2018-06-21 21:09:13,146 - INFO - launcher: Checking console encoding
[21.940946579] 2018-06-21 21:09:13,146 - INFO - launcher: Setting console encoding to UTF-8
[21.940946579] 2018-06-21 21:09:13,146 - INFO - launcher: Ensuring we're in the right environment
[22.939205170] 2018-06-21 21:09:13,147 - INFO - launcher: Adding local bins/ folder to path
[22.939205170] 2018-06-21 21:09:13,147 - INFO - launcher: Required checks passed.
[24.933099747] 2018-06-21 21:09:13,149 - INFO - launcher: Optional checks passed.
[24.933099747] 2018-06-21 21:09:13,149 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[0.7330393791198730] 2018-06-21 21:09:13,857 - WARNING - musicbot.config | In config.py::MainThread(12900), line 112 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[0.7330393791198730] 2018-06-21 21:09:13,857 - INFO - musicbot.config | In config.py::MainThread(12900), line 123 in run_checks: Using i18n: config/i18n/en.json
[0.7350339889526367] 2018-06-21 21:09:13,859 - DEBUG - musicbot.json | In json.py::MainThread(12900), line 8 in __init__: Init JSON obj with config/i18n/en.json
[0.8716685771942139] 2018-06-21 21:09:13,996 - DEBUG - musicbot.bot | In bot.py::MainThread(12900), line 227 in _setup_logging: Set logging level to INFO
[0.8716685771942139] 2018-06-21 21:09:13,996 - INFO - musicbot.bot | In bot.py::MainThread(12900), line 84 in __init__: Starting MusicBot 1.9.7
[0.8726658821105957] 2018-06-21 21:09:13,997 - INFO - musicbot.bot | In bot.py::MainThread(12900), line 90 in __init__: Loaded autoplaylist with 2531 entries
